### PR TITLE
Add env.BUILD_OPTS

### PR DESCRIPTION
**Pull request by ermalkaleci**
env.BUILD_OPTS will be passed to docker srtool run command

### CODE REVIEW

**Comment by xlc** [comment]
Can you verify if the resulting wasm is the same as the one from makefile command.

File: README.md
Comment (chevdor) [inline README.md:154]: This repo uses ASCIIDOC for the doc and the .md is generated so your change would be overriden.

Could you please add:
```
== Environmental variables

[source,bash]
----
    ...
    uses: chevdor/srtool-actions@v*
    env:
        # optional: will be passed to docker srtool run cmd
        BUILD_OPTS: --features on-chain-release-build
    with:
        ...
----
```

into `README_src.adoc` at L58 ?

You probably don't have the asciidoc toolchain so don't bother installing it, I will generate the new .md after merging.

File: action.yml
Comment (chevdor) [inline action.yml:173]: I would have preferred keeping it coherent here with either:
- `-e BUILD_OPTS=${{ env.BUILD_OPTS }}` like the rest
- or your admitedly more legible version everywhere 

I won't block the PR for that nonetheless as you went the safe route :)

**Review by chevdor — CHANGES_REQUESTED** [review]
Few small details but thanks a lot for the PR and sorry for the time it took to review. I mainly need those commits to be signed.

**Comment by chevdor** [comment]
As a sidenote, this PR bring the same option than the `--build-opts` flag of the [`srtool-cli`](https://github.com/chevdor/srtool-cli)

File: action.yml
Comment (ermalkaleci) [reply action.yml:173]: @chevdor `BUILD_OPTS` will have spaces and `=` so it will mess `CMD` when constructed. If I pass it as `-e BUILD_OPTS` it will read directly from env when running. I tried wrapping it with single quote but no luck. This was the only way I was able to get it running.

**Review by chevdor — APPROVED** [review]

**Comment by ermalkaleci** [comment]
@chevdor is there going to be a new tag?

**Comment by chevdor** [comment]
Yes but I want to update the doc and the examples first).

**Pull request merged**
